packages/server/.env-example

```diff
@@ -125,5 +125,3 @@ STRATEGY_LOCAL=true
 ############################################################
 # FRONTEND_HOST=127.0.0.1
 # FRONTEND_PORT=8081
-
-SPECKLE_AUTOMATE_URL="http://127.0.0.1:3030"
```

Comment:
Why remove it? There was still use in it being there for local envs

Reply:
This is the safeguard, to not enable overriding the automate url by default.